.github/issue-template.md

```diff
@@ -1,19 +1,14 @@
 ---
 name: Issue Template
 about: Template for both bug reports and feature requests
-
 ---
 
 ## Expected Behavior
 
-
 ## Actual Behavior
 
-
 ## Steps to Reproduce the Problem
 
-  1.
-  2.
-  3.
+1. 2. 3.
```

Comment:
well that's not good

@mattmoor i think you gotta make this thing ignore `.github`. or we need to put some sample lines in this

Reply:
It only seems to do this for some of them...  Weird.

Reply:
i think it's cuz there is no content in the list, so it's not recognizing it as one?